superset/commands/importers/v1/__init__.py

```diff
@@ -135,6 +138,8 @@ def _prevent_overwrite_existing_model(  # pylint: disable=invalid-name
         self, exceptions: List[ValidationError]
     ) -> None:
         """check if the object exists and shouldn't be overwritten"""
+        if self.config_overwrite:
+            return
         if not self.overwrite:
```

Comment:
We can simplify this to:

```suggestion
        if not self.overwrite and not self.config_overwrite:
```

Though maybe we should expand the logic a little bit. If `self.config_overwrite["dashboards'"]` is false, eg, we still want to check the UUIDs to see if the import would result in an overwrite, at least to catch the error early (it would fail later in the stack).

Maybe something like:

```python
        if not self.overwrite and not self.config_overwrite.get(f"{self.model_name}s"):
```

(But then maybe we should use singular form in `config_overwrite`, to align with the `model_name` attribute.)

Reply:
Hi @betodealmeida,  thank you for going over this PR.  
This PR aims to make the `"import modules flow"` more friendly, and configurability.  
While developing this feat I got into complexibility because I didn't want to break the flow others created.  

In the current `"import module flow"`, its works like this.  
1. upload the file.  
2. verify you don't overwrite existing models.  
3. if so, ask for permission, then overwrite the model.      

And about what you mentioned in the parenthesis ("it would fail later in the..") I didn't see it happening when I examen it.   
I comment on the validation part (on master) and upload a dashboards model (which contains dashboards that already exist) and I didn't get any errors.  indeed the import didn't overwrite any dashboard because the default is `overwrite==false`.   

In the  **new** `"import modules flow"` it will work like this.
1. Upload your file.
2. setup your configuration. and upload it.  

According to what's going now on the code,  
I saved backup capability to don't break the flow of validation.  

But if you will agree with me, I think that now, using the new flow we can delete the validate overwriting strategy, from both backend and frontend.

Reply:
So in short I don't see an added value in the `overwrite` param in the `import/module` API,   
after we can configure the overwrite